inc/namespace.php

```diff
@@ -116,6 +116,10 @@ function load_platform( $wp_debug_enabled ) {
 		load_db();
 	}
 
+	if ( $config['cloudfront-media-purge'] ) {
```

Comment:
I think maybe this should be `cdn-media-purge`? Better to not include the cloudfront product name